news/2015-11-25-arq-container-se-CR1.asciidoc

```diff
@@ -0,0 +1,19 @@
+---
+layout: news
+title: Arquillian container SE 1.0.0.CR1 released!
+author: Tomas Remes
+priority: 1
+change_frequency: weekly
+---
+
+We are pleased to announce new release of Arquillian container SE. If you are looking for reliable way to test your application in Java SE environment
```

Comment:
`to announce new release` -> `to announce the new release`
`for reliable way` ->  `for a reliable way`
`in Java SE environment` -> `in the Java SE`